fbchat/_file.py

```diff
@@ -252,6 +264,18 @@ def _from_subattachment(cls, data):
             uid=data["target"].get("video_id"),
         )
 
+    @classmethod
+    def _from_list(cls, data):
+        data = data["node"]
+        return cls(
+            width=data["original_dimensions"].get("x"),
+            height=data["original_dimensions"].get("y"),
+            small_image=data["image"],
+            medium_image=data["image1"],
+            large_image=data["image2"],
+            uid=data["id"],
```

Comment:
Use the attachment ID instead of the ID returned by the endpoint.

```suggestion
            uid=data["legacy_attachment_id"],
```